sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/joinReorder/JoinReorderSuite.scala

```diff
@@ -228,17 +228,16 @@ class JoinReorderSuite extends JoinReorderPlanTestBase with StatsEstimationTestB
   }
 
   test("SPARK-26352: join reordering should not change the order of attributes") {
```

Comment:
What happens in this test case's original test coverage? Is it preserved?
```
test("SPARK-26352: join reordering should not change the order of attributes") {
```

Reply:
I don't think the test coverage has changed. The intent of this test case is that check the project attributes should not change after ReorderJoin. In fact, after this PR, it also achieved this goal. also cc @rednaxelafx since you file this test case.

Reply:
Thanks for the ping, the fact that the expected plan did change its join order is good enough for the intent of the test. 👍

Reply:
Thank you for your confirmation, : )